### PR TITLE
doc: generate data-flow UML diagrams online

### PR DESCRIPTION
**Pull request by andriytk**
Note: the new `s3-{0,1,2}.txt` files were created from existing `index.rst`.

-----
[View rendered doc/data-flow/README.md](https://github.com/andriytk/cortx-motr/blob/gen-data-flow-uml/doc/data-flow/README.md)

### CODE REVIEW

File: doc/data-flow/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This is a sequence diagram for a typical cortx end-to-end data-flow.
+It shows processing of an S3 PUT request.
+
+![bb0](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/andriytk/cortx-motr/gen-data-flow-uml/doc/data-flow/s3-0.txt&fmt=svg)
```
Comment (huanghua78) [inline doc/data-flow/README.md:4]: This URL "https://raw.githubusercontent.com/andriytk/cortx-motr/gen-data-flow-uml/doc/data-flow/s3-0.txt" should be replaced by
"https://raw.githubusercontent.com/Seagate/cortx-motr/gen-data-flow-uml/doc/data-flow/s3-0.txt"
?

Comment (andriytk) [reply doc/data-flow/README.md:4]: Yes, after the landing.

Comment (huanghua78) [reply doc/data-flow/README.md:4]: Let's change it now.

Or we can commit the "s3-[0-2].txt" first and then commit this file with modified URL.
At that time, this file can be viewed properly.

Comment (andriytk) [reply doc/data-flow/README.md]: Done.

**Review by huanghua78 — APPROVED** [review]
Looks fine.

**Pull request merged**